config_v1.json

```diff
@@ -1,4 +1,4 @@
 {
   "packageName": "v1",
-  "packageVersion": "1.30.0"
+  "packageVersion": "1.32.0"
```

Comment:
Yes, config_v2.json should only have been changed in #19 but because I planned to merge that pull request immediately after this one, I left the change alone. Is that okay?